lib/bundles/inspec-compliance/cli.rb

```diff
@@ -56,6 +57,32 @@ def login(server) # rubocop:disable Metrics/AbcSize
       puts '', msg
     end
 
+    desc "automate SERVER --user='USER' --dctoken='DATA_COLLECTOR_TOKEN' or --usertoken='AUTOMATE_TOKEN' --ent='ENT'", 'Log in to an Automate SERVER'
```

Comment:
I am not 100% sure on this. What would be a better way to name it? Another option could be to just stick with `inspec compliance login` and just offer more options there. On the other hand this may become more complex for a user to understand which config is for which server? What are you thinking?